s3.tf

```diff
@@ -1,7 +1,32 @@
 resource "aws_s3_bucket" "demo-example" {
   bucket = "demoexample-1"
   versioning {
-    enabled = false
-    mfa_delete = false
+    enabled    = true
+    mfa_delete = true
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure versioning MFA is enabled for AWS S3 Buckets**
 
**Vulnerability**
Enabling MFA delete for versioning is a good way to add extra protection to sensitive files stored in AWS S3 Buckets.
 
**Remediation**
When working with S3 Versioning in Amazon S3 buckets, you can optionally add another layer of security by configuring a bucket to enable MFA (multi-factor authentication) delete. When you do this, the bucket owner must include two forms of authentication in any request to delete a version or change the versioning state of the bucket.
 In Terraform - 
 Set 'versioning.mfa_delete' attribute to 'true'.

source: accurics-devenv